app/services/app/app.ts

```diff
@@ -97,6 +97,19 @@ export class AppService extends StatefulService<IAppState>
 
       this.performanceMonitorService.start();
 
+      // Workaround for the browser source shutdown settings being not applied correctly
+      this.scenesService.getScenes().forEach(scene => {
+        scene.items.forEach(item => {
+          let isShowing: Boolean = true;
```

Comment:
you can remove the type declaration here, since you are assigning it immediately.  Anyway, `boolean` (lowercase) is the type you actually wanted.